sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/EventPosition.cs

```diff
@@ -16,26 +16,26 @@ namespace Azure.Messaging.EventHubs.Consumer
     public struct EventPosition : IEquatable<EventPosition>
     {
         /// <summary>The token that represents the beginning event in the stream of a partition.</summary>
-        private const string StartOfStreamOffset = "-1";
+        private const string StartingSequenceNumber = "-1";
 
         /// <summary>The token that represents the last event in the stream of a partition.</summary>
-        private const string EndOfStreamOffset = "@latest";
+        private const string LatestSequenceNumber = "@latest";
 
         /// <summary>
         ///   Corresponds to the location of the first event present in the partition.  Use this
         ///   position to begin receiving from the first event that was enqueued in the partition
         ///   which has not expired due to the retention policy.
         /// </summary>
         ///
-        public static EventPosition Earliest { get; } = FromOffset(StartOfStreamOffset, false);
+        public static EventPosition Earliest { get; } = FromSequenceNumber(StartingSequenceNumber, "", false);
 
         /// <summary>
         ///   Corresponds to the end of the partition, where no more events are currently enqueued.  Use this
         ///   position to begin receiving from the next event to be enqueued in the partition after an event
         ///   consumer begins reading with this position.
         /// </summary>
         ///
-        public static EventPosition Latest { get; } = FromOffset(EndOfStreamOffset, false);
+        public static EventPosition Latest { get; } = FromSequenceNumber("", "", true);
```

Comment:
I've tried a ton of different values here and can't find one that works properly and doesn't throw 😢